components/ws-daemon/pkg/content/initializer.go

```diff
@@ -234,7 +235,13 @@ func RunInitializerChild() (err error) {
 		return err
 	}
 
-	span := opentracing.StartSpan("RunInitializerChild", opentracing.FollowsFrom(tracing.FromTraceID(initmsg.TraceInfo)))
+	defer func() {
+		if err != nil {
```

Comment:
More of a general question but: is this kind of context access in a `defer`red function safe in Go?

Reply:
Yes. The deferred functions behave like any other closure/nested function as well. Because `error` is an interface type (read, a reference) we can just access it like this and don't need a pointer either.

Reply:
> The deferred functions behave like any other closure/nested function as well

Thanks!